src/components/deltabargraph.js

```diff
@@ -34,7 +46,7 @@ function DeltaBarGraph({timeseries, arrayKey}) {
         0,
         Math.max(
           1,
-          d3.max(data, (d) => d[arrayKey])
+          d3.max(data, (d) => Math.abs(d[arrayKey]))
         ),
```

Comment:
You'll have to handle the active chart separately here since it's domain can contain values less than 0.